src/Components/Header.js

```diff
@@ -51,12 +51,14 @@ function Header({setQuery}) {
               />
             </div>
             <div className="links">
+
               <ul data-testid="header_links">
-                <li data-testid="header_sub" onClick={() => window.open('http://tiny.cc/new-project', '_blank')}>Submit Project</li>
+                {loggedin && <li data-testid="header_sub" onClick={goTo('submit-project')}>Submit Project</li>}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/CSC510-Group-25/feature-hunt/pullRequest?prid=8358184)
```suggestion
                {loggedin && <li data-testid="header_sub" onClick={goTo("submit-project")}>Submit Project</li>}
```